### PR TITLE
updated doc

### PR DESCRIPTION
**Pull request by rishu2403**

### CODE REVIEW

File: colosseum/colors.py [hunk 1]
```diff
@@ -1,3 +1,56 @@
+'''HTML colors are broken into 3 components:
+
+R: Red
+G: Green
+B: Blue
+
+These 3 components may be represented in hexadecimal.
```
Comment (brutusthebee) [inline colosseum/colors.py]: At column 54: (W291) trailing whitespace

File: colosseum/colors.py [hunk 2]
```diff
@@ -1,3 +1,56 @@
...
+In decimal, the 2nd digit from the right are the tens.
+In hexadecimal, however, these are the 16s.
+
+Each component is a two-digit hexadecimal value.
+0x00 is equivalent to 0:
+  0 * 1   = 0
++ 0 * 16  = 0
+-------------
+          = 0
+
```
Comment (brutusthebee) [inline colosseum/colors.py]: At column 1: (W293) blank line contains whitespace

File: colosseum/colors.py [hunk 3]
```diff
@@ -1,3 +1,56 @@
...
+-------------
+          = 0
+          
+0xFF is equivalent to 255:
+  15 * 16 = 240
++ 15 * 1  =  15
+---------------
+          = 255
+
+From this simple conversion, we can easily figure out the total number of possible colors.
```
Comment (brutusthebee) [inline colosseum/colors.py]: At column 91: (W291) trailing whitespace

File: colosseum/colors.py [hunk 4]
```diff
@@ -1,3 +1,56 @@
...
+              F              F             F            F           F          F
+   15 * 1048576  = 15728640
+   15 * 65536    =   983040
+   15 * 4096     =    61440
+   15 * 256      =     3840
+   15 * 16       =      240
++  15 * 1        =       15
+---------------------------
+                 = 16777215 color possibilities
+
```
Comment (brutusthebee) [inline colosseum/colors.py]: At column 1: (W293) blank line contains whitespace

File: colosseum/colors.py [hunk 5]
```diff
@@ -1,3 +1,56 @@
...
+   15 * 4096     =    61440
+   15 * 256      =     3840
+   15 * 16       =      240
++  15 * 1        =       15
+---------------------------
+                 = 16777215 color possibilities
+ 
+'''
 
 class Color:
```
Comment (brutusthebee) [inline colosseum/colors.py]: At column 1: (E302) expected 2 blank lines, found 1

File: colosseum/colors.py (hunk 1 above)
Comment (brutusthebee) [inline colosseum/colors.py]: At column 54: (W291) trailing whitespace

File: colosseum/colors.py [hunk 6]
```diff
@@ -1,3 +1,56 @@
...
+In decimal, the 2nd digit from the right are the tens.
+In hexadecimal, however, these are the 16s.
+
+Each component is a two-digit hexadecimal value.
+0x00 is equivalent to 0:
+  0 * 1   = 0
++ 0 * 16  = 0
+-------------
+= 0
+
```
Comment (brutusthebee) [inline colosseum/colors.py]: At column 1: (W293) blank line contains whitespace

File: colosseum/colors.py [hunk 7]
```diff
@@ -1,3 +1,56 @@
...
+-------------
+= 0
+          
+0xFF is equivalent to 255:
+  15 * 16 = 240
++ 15 * 1  =  15
+---------------
+= 255
+
+From this simple conversion, we can easily figure out the total number of possible colors.
```
Comment (brutusthebee) [inline colosseum/colors.py]: At column 91: (W291) trailing whitespace

File: colosseum/colors.py [hunk 8]
```diff
@@ -1,3 +1,56 @@
...
+              F              F             F            F           F          F
+   15 * 1048576  = 15728640
+   15 * 65536    =   983040
+   15 * 4096     =    61440
+   15 * 256      =     3840
+   15 * 16       =      240
++  15 * 1        =       15
+---------------------------
+= 16777215 color possibilities
+
```
Comment (brutusthebee) [inline colosseum/colors.py]: At column 1: (W293) blank line contains whitespace

File: colosseum/colors.py [hunk 9]
```diff
@@ -1,3 +1,56 @@
...
+   15 * 4096     =    61440
+   15 * 256      =     3840
+   15 * 16       =      240
++  15 * 1        =       15
+---------------------------
+= 16777215 color possibilities
+ 
+'''
 
 class Color:
```
Comment (brutusthebee) [inline colosseum/colors.py]: At column 1: (E302) expected 2 blank lines, found 1

File: colosseum/colors.py [hunk 10]
```diff
@@ -1,3 +1,47 @@
+'''HTML colors are broken into 3 components:
+
+R: Red
+G: Green
+B: Blue
+These 3 components may be represented in hexadecimal.
```
Comment (brutusthebee) [inline colosseum/colors.py]: At column 54: (W291) trailing whitespace

File: colosseum/colors.py [hunk 11]
```diff
@@ -1,3 +1,47 @@
...
+  0 * 1   = 0
++ 0 * 16  = 0
+-------------
+= 0
+0xFF is equivalent to 255:
+  15 * 16 = 240
++ 15 * 1  =  15
+---------------
+= 255
+From this simple conversion, we can easily figure out the total number of possible colors.
```
Comment (brutusthebee) [inline colosseum/colors.py]: At column 91: (W291) trailing whitespace

File: colosseum/colors.py [hunk 12]
```diff
@@ -1,3 +1,47 @@
...
+           F              F             F            F           F          F
+   15 * 1048576  = 15728640
+   15 * 65536    =   983040
+   15 * 4096     =    61440
+   15 * 256      =     3840
+   15 * 16       =      240
++  15 * 1        =       15
+---------------------------
+= 16777215 color possibilities
+
```
Comment (brutusthebee) [inline colosseum/colors.py]: At column 1: (W293) blank line contains whitespace

File: colosseum/colors.py [hunk 13]
```diff
@@ -1,3 +1,47 @@
...
+   15 * 4096     =    61440
+   15 * 256      =     3840
+   15 * 16       =      240
++  15 * 1        =       15
+---------------------------
+= 16777215 color possibilities
+ 
+'''
 
 class Color:
```
Comment (brutusthebee) [inline colosseum/colors.py]: At column 1: (E302) expected 2 blank lines, found 1

File: colosseum/colors.py [hunk 14]
```diff
@@ -1,3 +1,39 @@
+'''HTML colors are broken into 3 components:
+R: Red
+G: Green
+B: Blue
+These 3 components may be represented in hexadecimal.
```
Comment (brutusthebee) [inline colosseum/colors.py]: At column 54: (W291) trailing whitespace

File: colosseum/colors.py [hunk 15]
```diff
@@ -1,3 +1,39 @@
...
+In decimal, the 2nd digit from the right are the tens.
+In hexadecimal, however, these are the 16s.
+Each component is a two-digit hexadecimal value.
+0x00 is equivalent to 0:
+0 * 1   = 0
++0 * 16  = 0= 0
+0xFF is equivalent to 255:
+15 * 16 = 240
++15 * 1  =  15= 255
+From this simple conversion, we can easily figure out the total number of possible colors.
```
Comment (brutusthebee) [inline colosseum/colors.py]: At column 91: (W291) trailing whitespace

File: colosseum/colors.py [hunk 16]
```diff
@@ -1,3 +1,39 @@
...
+           F              F             F            F           F          F
+15 * 1048576=15728640
+15 * 65536=983040
+15 * 4096=61440
+15 * 256=3840
+15 * 16=240
++15 * 1=15
+=16777215 color possibilities'''
 
 class Color:
```
Comment (brutusthebee) [inline colosseum/colors.py]: At column 1: (E302) expected 2 blank lines, found 1

File: colosseum/colors.py [hunk 17]
```diff
@@ -1,3 +1,39 @@
+'''HTML colors are broken into 3 components:
+R:Red
+G:Green
+B:Blue-These 3 components may be represented in hexadecimal.
```
Comment (brutusthebee) [inline colosseum/colors.py]: At column 61: (W291) trailing whitespace

File: colosseum/colors.py [hunk 18]
```diff
@@ -1,3 +1,39 @@
...
+In decimal, the 2nd digit from the right are the tens.
+In hexadecimal, however, these are the 16s.
+Each component is a two-digit hexadecimal value.
+0x00 is equivalent to 0:
+0*1= 0
++0*16=0=0
+0xFF is equivalent to 255:
+15*16=240
++15*1=15=255
+From this simple conversion, we can easily figure out the total number of possible colors.
```
Comment (brutusthebee) [inline colosseum/colors.py]: At column 91: (W291) trailing whitespace

File: colosseum/colors.py [hunk 19]
```diff
@@ -1,3 +1,36 @@
+'''HTML colors are broken into 3 components:
+R:Red G:Green B:Blue-These 3 components may be represented in hexadecimal.
```
Comment (brutusthebee) [inline colosseum/colors.py]: At column 75: (W291) trailing whitespace

File: colosseum/colors.py [hunk 20]
```diff
@@ -1,3 +1,36 @@
...
+Likewise in hexadecimal.
+In decimal, the 2nd digit from the right are the tens.
+In hexadecimal, however, these are the 16s.
+Each component is a two-digit hexadecimal value.
+0x00 is equivalent to 0:
+0*1= 0
++0*16=0=0
+0xFF is equivalent to 255:
+15*16=240
++15*1=15=255 From this simple conversion, we can easily figure out the total number of possible colors.
```
Comment (brutusthebee) [inline colosseum/colors.py]: At column 104: (W291) trailing whitespace

File: colosseum/colors.py [hunk 21]
```diff
@@ -1,3 +1,33 @@
+'''
+
+HTML colors are broken into 3 components:
+R:Red G:Green B:Blue-These 3 components may be represented in hexadecimal.
```
Comment (brutusthebee) [inline colosseum/colors.py]: At column 75: (W291) trailing whitespace

File: colosseum/colors.py [hunk 22]
```diff
@@ -1,3 +1,33 @@
...
+Likewise in hexadecimal.
+In decimal, the 2nd digit from the right are the tens.
+In hexadecimal, however, these are the 16s.
+Each component is a two-digit hexadecimal value.
+0x00 is equivalent to 0:
+0*1= 0
++0*16=0=0
+0xFF is equivalent to 255:
+15*16=240
++15*1=15=255 From this simple conversion, we can easily figure out the total number of possible colors.
```
Comment (brutusthebee) [inline colosseum/colors.py]: At column 104: (W291) trailing whitespace

File: colosseum/colors.py [hunk 23]
```diff
@@ -1,3 +1,26 @@
+'''HTML colors are broken into 3 components: R:Red G:Green B:Blue-These 3 components may be represented in hexadecimal.
```
Comment (brutusthebee) [inline colosseum/colors.py]: At column 120: (W291) trailing whitespace

File: colosseum/colors.py [hunk 24]
```diff
@@ -1,3 +1,26 @@
...
+White:  #FFFFFF 0xFF, 0xFF, 0xFF
+In decimal, the digits are 0-9.
+In hexadecimal, the digits are 0-F, which is equivalent to 0-15 in decimal.
+In decimal, the rightmost digit are the ones.
+Likewise in hexadecimal.
+In decimal, the 2nd digit from the right are the tens.
+In hexadecimal, however, these are the 16s.
+Each component is a two-digit hexadecimal value.
+0x00 is equivalent to 0: 0*1= 0 +0*16=0=0 0xFF is equivalent to 255: 15*16=240
++15*1=15=255 From this simple conversion, we can easily figure out the total number of possible colors.
```
Comment (brutusthebee) [inline colosseum/colors.py]: At column 104: (W291) trailing whitespace

File: colosseum/colors.py [hunk 25]
```diff
@@ -1,3 +1,25 @@
...
```
Comment (brutusthebee) [inline colosseum/colors.py]: At column 104: (W291) trailing whitespace

File: colosseum/colors.py [hunk 26]
```diff
@@ -1,3 +1,20 @@
+'''A hexadecimal representation of the 3 components is called a triplet, or group of 3 numbers.R G B    R     G     B
+Black:  #000000 0x00, 0x00, 0x00
+Red:    #FF0000 0xFF, 0x00, 0x00
+Green:  #00FF00 0x00, 0xFF, 0x00
+Blue:   #0000FF 0x00, 0x00, 0xFF
+White:  #FFFFFF 0xFF, 0xFF, 0xFF
+In decimal, the digits are 0-9.In hexadecimal, the digits are 0-F, which is equivalent to 0-15 in decimal.
+In decimal, the rightmost digit are the ones.Likewise in hexadecimal.In decimal, the 2nd digit from the right are the tens.In hexadecimal, however, these are the 16s.
```
Comment (brutusthebee) [inline colosseum/colors.py]: At column 120: (E501) line too long (166 > 119 characters)

File: colosseum/colors.py [hunk 27]
```diff
@@ -1,3 +1,20 @@
...
+Each component is a two-digit hexadecimal value.0x00 is equivalent to 0: 0*1= 0 +0*16=0=0 0xFF is equivalent to 255: 15*16=240
```
Comment (brutusthebee) [inline colosseum/colors.py]: At column 120: (E501) line too long (126 > 119 characters)

File: colosseum/colors.py [hunk 28]
```diff
@@ -1,3 +1,20 @@
...
++15*1=15=255 From this simple conversion, we can easily figure out the total number of possible colors.
```
Comment (brutusthebee) [inline colosseum/colors.py]: At column 104: (W291) trailing whitespace

**Comment by phildini** [comment]
I don't think this PR does what you meant for it to do, and it's been about a month since it was touched, so I'm going to close it for now. Feel free to re-open if you want to keep it going.

**Pull request closed without merging**